.azure_pipelines/pytorch-tests-pipeline.yml

```diff
@@ -0,0 +1,37 @@
+# PyTorch PR PyTorch Tests Builds Pipeline on Azure DevOps
+#
+# This pipeline runs custom PyTorch unit-tests on PyTorch 
+# wheels generated during PR builds.
+
+stages:
+- stage: 'PRCustomTests'
+  displayName: 'Run custom unit tests on PyTorch wheels'
+  jobs:
+  - template: job_templates/build-verify-publish-template-unix.yml
+    parameters:
+      name: ubuntu_1804_CPU_docker
```

Comment:
are we going to run PR validation on both linux CPU and GPU? I used to believe we would only run on GPU?

Reply:
Currently, we are only planning to do PR validation on Linux GPU. The Linux CPU logic is there temporarily as sort of a "baseline" for the Linux CPU runs. The CPU runs are also less complicated, and it gives me an idea of how the PR Build logic would be with CPUs. I'll be removing the CPU PR builds before making a PR to `pytorch/pytorch`.

Reply:
How do you plan to keep track of all the changes that need to be made submitting the PR on `pytorch/pytorch` ? :-)

Reply:
Haha :D All the changes in this PR, barring the Dockerfile change in Docker/pytorch/ubuntu_cpu_gpu, are in the `.azure_pipelines` folder. A simple delete of the current `.azure_pipelines/` and upload of this new `.azure_pipelines/` folder will do the trick.